### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by pinussilvestrus**
* Use `files` over `.npmignore`
* Bump dev dependencies in library and example
* Bootstrap example on initial setup
* Test against `node@16`

### CODE REVIEW

**Comment by pinussilvestrus** [comment]
Did the cleanups while investigating https://github.com/bpmn-io/react-bpmn/issues/14.

File: package.json [hunk 1]
```diff
@@ -3,10 +3,16 @@
...
+    "dist",
+    "src"
+  ],
   "scripts": {
     "all": "npm run bundle",
     "bundle": "NODE_ENV=production rollup -c",
-    "prepublishOnly": "npm run bundle"
+    "prepublishOnly": "npm run bundle",
+    "postinstall": "cd example && npm install",
+    "start:example": "cd example && npm start"
```
Comment (marstamm) [inline package.json]: As we don't have an example, we could use the `start` script directly
```suggestion
    "start": "cd example && npm start"
```

File: README.md [hunk 2]
```diff
@@ -36,6 +36,15 @@ function App(props) {
...
 
+## Development
+
+To get the development setup make sure to have [NodeJS](https://nodejs.org/en/download/) installed.
+As soon as you are set up, clone the project and execute
+
+```
+npm install
+npm run all
+```
```
Comment (marstamm) [inline README.md:47]: I would also add a quick note on the example, as it is not obvious that you need to open a sub-directory:
```
## Example

To run the example, execute `npm run start:example` and open http://localhost:5000/public in your Webbrowser.
```

What do you think?

Comment (pinussilvestrus) [reply README.md:47]: Thanks! I added this via https://github.com/bpmn-io/react-bpmn/pull/15/commits/2161ee5933dc12f5502baf69dafe4410d517da54

**Review by marstamm — APPROVED** [review]
🚀

**Pull request merged**